db/migrate/20231103060018_fix_invalid_topic_user_bookmarked_data.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class FixInvalidTopicUserBookmarkedData < ActiveRecord::Migration[7.0]
+  def up
+    DB.exec(<<~SQL)
+      UPDATE topic_users
+      SET bookmarked = true
+      WHERE id IN (
+        SELECT
+          topic_users.id
+        FROM topic_users
+        INNER JOIN bookmarks ON bookmarks.user_id = topic_users.user_id
+        WHERE bookmarkable_type = 'Topic'
+          AND bookmarks.bookmarkable_id = topic_users.topic_id
+          AND topic_users.bookmarked = false
+      ) AND topic_users.bookmarked = false
```

Comment:
Just to check, bookmarkables only exist in the table if there is an active bookmark, and is deleted when the bookmark is gone, right? e.g. no "bookmark: true"

Reply:
@nattsw yes that's right, to get rid of a bookmark we delete the Bookmark record :)